### PR TITLE
Add 'year' & 'runs' optional params + README improvement

### PR DESCRIPTION
**Pull request by Matbabs**
Your work is really great! 

I took the liberty of adding 2 optional parameters:

-`year` (by default the current year is retrieved)
-`runs` (by default 200 as you wrote, but I let the choice)

Moreover I improved a bit the README for the installation and the use.

**The addition of the year allows to test on the old events !**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,58 @@
...
+# Table of Contents
+
+- [How it works](#how-it-works)
+- [How to use](#how-to-use)
+- [Installation](#installation)
+
+# How it works
+
+The code is written in Python, and uses the OpenAI API to call GPT-3.
+It also uses the `aoc` library to download the puzzle input and submit the answer.
```
Comment (max-sixty) [inline README.md:18]:
```suggestion
It also uses the `aoc-cli` library to download the puzzle input and submit the answer.
```

And feel free to add a link

File: README.md [hunk 2]
```diff
@@ -3,5 +3,58 @@
...
+* `--year` - The year of the puzzle. Defaults to the current year.
+* `--part` - The part of the puzzle.
+* `--n-workers` - The number of workers to use. Defaults to 1
+* `--runs` - The number of runs to make. Defaults to 200
+* `--stop-when-submitted` - Stop when the answer is submitted. Defaults to False
+
+Example:
+
+```bash
+$ python3 openai.py --day 2 --year 2019 --runs 10
```
Comment (max-sixty) [inline README.md:38]:
```suggestion
$ python3 openai.py --day=2 --year=2019 --runs=10
```

File: README.md [hunk 3]
```diff
@@ -3,5 +3,58 @@
...
+
+# How it works
+
+The code is written in Python, and uses the OpenAI API to call GPT-3.
+It also uses the `aoc` library to download the puzzle input and submit the answer.
+
+# How to use
+
+```bash
+$ python3 openai.py --day 2
```
Comment (max-sixty) [inline README.md:23]: nit, feel free to discard
```suggestion
$ python3 openai.py --day=2
```

File: README.md [hunk 4]
```diff
@@ -3,5 +3,61 @@
...
+    # winget install aoc-cli
+
+
+> Different Advent of Code users get different puzzle input. To download your input and submit your answer, you need an adventofcode.com session cookie. To obtain your session cookie, login to the Advent of Code website and inspect the session value of the cookie that gets stored in your browser. Put the session number (a long hex string) in a file called .adventofcode.session in your home directory. This file should only contain your session number, in a single line.
+
+Install the `openai` library:
+
+    pip install openai
+
+> Add your OpenAI API key to the `OPENAI_API_KEY` environment variable.
```
Comment (max-sixty) [inline README.md:62]: And someone needs to add the AOC API key as per the instructions in `aoc-cli` (a link is great)

**Review by max-sixty — APPROVED** [review]
Thanks @Matbabs!

File: README.md (hunk 1 above)
Comment (Matbabs) [reply README.md]: Done !

File: README.md (hunk 2 above)
Comment (Matbabs) [reply README.md]: Done !

File: README.md (hunk 3 above)
Comment (Matbabs) [reply README.md]: Done !

File: README.md (hunk 4 above)
Comment (Matbabs) [reply README.md:73]: Done !

Comment (max-sixty) [reply README.md:73]: And sorry, you already had the explanation in the quoted section!

**Pull request merged**